r/tests/testthat/test-matrix_stats.R

```diff
@@ -190,6 +190,7 @@ test_that("svds registers generic with RSpectra", {
 })
 
 test_that("rowMaxs and colMaxs works comprehensive", {
+  skip_if_not_installed("matrixStats")
```

Comment:
As we will be growing the amount of functions that utilize matrixStats interfaces, I would suggest doing an install of `matrixStats` within the workflow itself.  It is already in the `suggests` within the R description

Reply:
The `r-lib/actions/setup-r-dependencies@v2` does already install all the `suggests` dependencies, so these will run. This change was just a drive-by change tangentially related to this PR.